### PR TITLE
Rubocop fixes

### PR DESCRIPTION
**Pull request by bpollack**
These are all leading whitespace fixes. While the number of lines changed looks scary, if you turn on "ignore whitespace," basically the whole thing goes away.

### CODE REVIEW

File: lib/active_merchant/billing/gateways/eway_managed.rb [hunk 1]
```diff
@@ -232,33 +232,33 @@ def soap_request(arguments, action)
...
         xml = Builder::XmlMarkup.new :indent => 2
-          xml.instruct!
-          xml.tag! 'soap12:Envelope', {'xmlns:xsi' => 'http://www.w3.org/2001/XMLSchema-instance', 'xmlns:xsd' => 'http://www.w3.org/2001/XMLSchema', 'xmlns:soap12' => 'http://www.w3.org/2003/05/soap-envelope'} do
-            xml.tag! 'soap12:Header' do
-              xml.tag! 'eWAYHeader', {'xmlns' => 'https://www.eway.com.au/gateway/managedpayment'} do
-                xml.tag! 'eWAYCustomerID', @options[:login]
-                xml.tag! 'Username', @options[:username]
-                xml.tag! 'Password', @options[:password]
-              end
+        xml.instruct!
```
Comment (nfarve) [inline lib/active_merchant/billing/gateways/eway_managed.rb]: Shouldn't this be indented?

File: lib/active_merchant/billing/gateways/paystation.rb [hunk 2]
```diff
@@ -100,101 +100,101 @@ def scrub(transcript)
 
       private
 
-        def new_request
```
Comment (nfarve) [inline lib/active_merchant/billing/gateways/paystation.rb]: Did most of this file get re-ordered?

File: lib/active_merchant/billing/gateways/eway_managed.rb (hunk 1 above)
Comment (bpollack) [reply lib/active_merchant/billing/gateways/eway_managed.rb]: No; there's no `do` on the previous line, and this is not a block. It's honestly really misleading for exactly that reason as-is.

File: lib/active_merchant/billing/gateways/paystation.rb (hunk 2 above)
Comment (bpollack) [reply lib/active_merchant/billing/gateways/paystation.rb]: No. I'm not honestly entirely clear what you're responding to; the lines match up one-for-one.

Comment (bpollack) [reply lib/active_merchant/billing/gateways/paystation.rb]: Oh, I see, GitHub's being dumb in side-by-side mode. Either turn on "ignore whitespace" or use a different diff viewer. Or you can look at the actual line numbers.

**Pull request merged**